### DISCUSSION TITLE
Help using lists

### DISCUSSION THREAD

**Discussion by blogdemoi** [iptv-org/community#81] [Q&A] [answered]
Hi,

I am using vlc on my android 12 phone and need help.
I copy/paste the French list in the new feed section in vlc, it plays the forget TV channel OK. But how can I have a list of the TV channels in the French list feeds, so I can easily navigate and browse them...then play them.

Thank you.

**Comment by benevenstanciano** [comment]
The app Mega IPTV is more user-friendly.

**Comment by BellezaEmporium** [comment, marked as answer]
FR : Il est un peu difficile de vous comprendre concernant le souci que vous rencontrez avec nos playlists. Vous nous dites que vous n'avez pas accès à toutes les chaînes quand vous faites un copier/coller du lien de la playlist française sur VLC ? Si c'est le cas, regardez si vous avez la liste de lecture activée, sur l'onglet Vue->Liste de lecture intégrée.

EN : It is a bit difficult to understand you concerning the issue you face with our playlists. You're saying that do you do not have access to all the channels when you make a copy and paste of the French playlist link on VLC ? If that's the case, make sure you have the playlist view active, on the View>Docked Playlist navbar.

**Comment by blogdemoi** [comment]
Hi,
Thank you for your help. My problem is solved. I can now see the playlist and TV channels and browse them. I didn't know how to show them, now it is OK.
